### DISCUSSION TITLE
MAA启动慢。

### DISCUSSION THREAD

**Discussion by testpaper007** [MaaAssistantArknights/MaaAssistantArknights#6302] [Q&A]
MAA启动缓慢，多次双击才能运行，管理员模式也是一样。
同时频繁出现“资源损坏”的弹窗，关闭后重启恢复正常。（已经重新下载过一次了还是一样的问题。）

![K1%`VQE)TC@R V914}XFJYG](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/139633765/3bb54c44-bd90-48cb-9269-c21be4000a7e)

**Comment by zzyyyl** [comment]
更新一下
